lib/rules/require-parentheses-around-arrow-param.js

```diff
@@ -48,6 +48,9 @@ module.exports.prototype = {
     check: function(file, errors) {
         function isWrapped(node) {
             var openParensToken = file.getPrevToken(file.getFirstNodeToken(node));
+            if (openParensToken.value === '...') {
```

Comment:
This is a drastically different condition and the test seemingly didn't have to change. Something is fishy about that.